pvm.py

```diff
@@ -43,9 +43,13 @@ def parse_commands(*args):
 		pass
 	# TODO: Create another python file to control two display
 	if command=="file":
```

Comment:
it seems that we are using the `PREFIX_PATH` more than we should be using it, and that `VIDEO_PATH` is more of a `VIDEO_FILE` rather than the full path to that video file. So, I'd suggest starting this here by defining `VIDEO_PATH` and then just using it. Please correct me if you can see some advantages for not doing that.
```suggestion
	if command=="file":
		VIDEO_PATH = PREFIX_PATH + value
```

Reply:
Changed.